lib/storage/providers/SQLiteStorage.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * The SQLiteStorage provider stores everything in a key/value store by
+ * converting the value to a JSON string
+ */
+import {open} from 'react-native-quick-sqlite';
+import _ from 'underscore';
+
+const DB_NAME = 'Expensify-new-db';
+const db = open({name: DB_NAME});
+
+db.execute('CREATE TABLE IF NOT EXISTS magic_map (record_key TEXT NOT NULL PRIMARY KEY , valueJSON JSON NOT NULL) WITHOUT ROWID;');
```

Comment:
Have we settled on this name for the DB?

Reply:
No, I had to pass a name and it was first what crossed my mind :). What would be a better name?

Reply:
hmmmm how about `keyvaluepairs`?

Reply:
I agree with @iwiznia 
Since `react-native-onyx` is an utility that can be used in any project and not specifically "New Expensify" the name should be more generic

BTW for web (IndexedDB) we use the name `OnyxDB` 

https://github.com/Expensify/react-native-onyx/blob/91aabb7219ddc8722cc2955de0398b1ae70c546d/lib/storage/providers/LocalForage.js#L12-L14

Reply:
Let's change it